### ISSUE TITLE
Add User Identification Feature

### ISSUE THREAD

**Issue by Dr0p42** [jupyter-naas/abi#284]
A user has asked for their name, indicating a need for a feature that can identify and recall user-specific information such as their name. This feature would enhance user interaction by providing personalized responses. The system currently lacks the ability to remember or identify individual users, which could be beneficial for creating a more personalized experience.

**Comment by jravenel** [comment]
I agree with this and I'm thinking on incorporating AIA back as /custom module for the user in ABI. Will take this task next.

**Comment by jravenel** [comment]
Resources: 
- https://www.youtube.com/watch?app=desktop&v=Z_oWzTuljss
- https://www.youtube.com/watch?v=3PIqhdRzhxE
- https://www.youtube.com/watch?v=Wqf2GimAlWo
